collector/container/rhtap-slim.Dockerfile

```
@@ -0,0 +1,151 @@
+ARG BUILD_DIR=/build
+ARG SRC_ROOT_DIR=${BUILD_DIR}
+ARG CMAKE_BUILD_DIR=${SRC_ROOT_DIR}/cmake-build
+ARG USE_VALGRIND=false
+ARG ADDRESS_SANITIZER=false
+
+# TODO(ROX-20312): we can't pin image tag or digest because currently there's no mechanism to auto-update that.
+# TODO(ROX-20651): Use RHEL/ubi base image when entitlement is solved.
+# RPMs requiring entitlement: bpftool, cmake-3.18.2-9.el8, elfutils-libelf-devel, tbb-devel, c-ares-devel, jq-devel
+# FROM registry.access.redhat.com/ubi8/ubi:latest AS builder
+FROM quay.io/centos/centos:stream9 AS builder
+
+COPY . .
+
+# TODO(ROX-20234): use hermetic builds when installing/updating RPMs becomes hermetic.
+RUN dnf -y update \
```

TIL that `dnf update` is the same as `dnf upgrade`.

```suggestion
RUN dnf upgrade -y --nobest \
```

See related https://gitlab.cee.redhat.com/stackrox/rhacs-midstream/-/merge_requests/2263